Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -135,6 +135,8 @@
       - id: BoxPDAEngineering # Delta-V
       - id: RCD
       - id: RCDAmmo
+      - id: RPD
+      - id: RCDAmmo #Extra Compressed matter for the RPD
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)